x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/csp_boxed_radio_group.tsx

```diff
@@ -33,16 +34,30 @@ export const InlineRadioGroup = ({ idSelected, size, options, disabled, onChange
       onChange={onChange}
       css={css`
         display: grid;
-        grid-template-columns: repeat(${options.length}, 1fr);
-        grid-template-rows: ${size === 's' ? euiTheme.size.xxl : euiTheme.size.xxxl};
-        column-gap: ${euiTheme.size.s};
         align-items: center;
 
+        // Show columns for m+ screens (above 768px)
+        @media only screen and (min-width: ${euiTheme.breakpoint.m}px) {
```

Comment:
added media query to prevent it from breaking on mobile:

![Screen Shot 2022-12-26 at 14 40 56](https://user-images.githubusercontent.com/20814186/209556445-fd1eb1ea-bf00-4788-84be-fdaf19e6eb60.png)

Reply:
as a side note - maybe the `Setup Access` bit section should be before the radio group? 
![Screen Shot 2022-12-26 at 14 40 06](https://user-images.githubusercontent.com/20814186/209556596-6d296b5f-61a0-4a7b-9bf7-3173d488cd01.png)

Reply:
dont think so
![image](https://user-images.githubusercontent.com/51442161/209565492-e1036cc9-51dd-4227-8acb-0267ecf9116b.png)

Eventually we want to get this result, the structure of `title > description > inputs` is the way to go, and looks much better

Reply:
good catch! 

changed to include the title and some of the description (as we're not implementing all of the design now, like editing name/description)

# CSPM
![Screen Shot 2022-12-27 at 9 29 10](https://user-images.githubusercontent.com/20814186/209628939-ff253a2e-7862-4859-b26b-da816c8a18f6.png)

# KSPM 
![Screen Shot 2022-12-27 at 9 28 53](https://user-images.githubusercontent.com/20814186/209628969-2669cfa8-8012-4d87-98fa-bf4010a2df03.png)


i think the CSPM text can be improved. no need for `(CSP)`